cmd/kubelet/app/server_bootstrap_test.go

```diff
@@ -254,7 +255,11 @@ func (s *csrSimulator) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	defer s.lock.Unlock()
 	t := s.t
 
-	t.Logf("Request %s %s %s", req.Method, req.URL, req.UserAgent())
+	// filter out timeouts as csrSimulator don't support them
+	req.URL.RawQuery = regexp.MustCompile("&timeout=[0-9smh]*").ReplaceAllLiteralString(req.URL.RawQuery, "")
```

Comment:
The * at the end means that we can probably filter out some other parameters.
Maybe that's fine, but if so this requires updating the comment.

Reply:
the `*` applies only to the set the it is following so it shouldn't filter out other params - https://play.golang.org/p/eOMx58K6U2x

Reply:
suggest
```go
q := req.URL.Query()
q.Del("timeout")
q.Del("timeoutSeconds")
req.URL.RawQuery = q.Encode()
```